### ISSUE TITLE
CFC: Merge Web Workers into HTML

### ISSUE THREAD

**Issue by LJWatson** [w3c/html#1090]
This is a Call For Consensus (CFC) to merge the [Web Workers](https://w3c.github.io/workers/) specification into the [HTML](https://w3c.github.io/html) specification.

The reason for merging the two specifications is that it would make it easier to maintain Web Workers, and therefore more likely that the Web Workers parts of the HTML specification will be kept up to date (and issues addressed more responsively).

The proposal was raised as issue #1075, and Sangwhan Moon (the current Web Workers editor) has [agreed to do the work](https://github.com/w3c/html/issues/1075#issuecomment-347460911).

Please respond to this CFC by the end of day on Thursday 7th December. To support the proposal, add a "thumbs up" to this comment. If you don't support the proposal, add a "thumbs down" and post your reasons in a comment.

If you choose not to respond it will be taken as silent support for the proposal. Actual responses are preferred however.

**Comment by awwright** [comment]
I work on some other Web technologies that don't touch HTML, like Semantic Web and JSON Schema.

I understand there may be editorial reasons and prior precedent for things like this, however I tend to disagree, as I think this presents a very HTML-centric view of the Web. There's no part of Web Workers that depends on HTML. This would significantly complicate moves to use Web Workers in other technologies, like server side, or applications like SVG.

In short, HTML is a markup language, not an operating system. It would benefit people who write user agents if things not relating to rendering a DOM structure or parsing HTML documents were moved into their own TR.

**Comment by LJWatson** [comment]
Thanks for your comment @awwright.

This is one of those times where there isn't a right or wrong answer, just different approaches with different priorities. In this instance spec maintenance has been prioritised, on the basis that a spec that needs updating is no help to anyone no matter whether it's on /TR in its own right or not.

**Comment by LJWatson** [comment]
This CFC received several positive responses, and one expression of concern (from outside the WebPlat WG). We have noted the concern and responded, but the overall consensus is to adopt the proposed plan.

This CFC therefore passes. Thanks to everyone who responded.

**Comment by chaals** [comment]
Looking again, and at the logistics of maintaining this specification as part of HTML, I think @awwright is correct, and that we (including me as one of those who voted for it), reached the wrong decision.

In part, the goal of making the spec easier to maintain has not proven to be achieved by this approach, which begins by adding a substantial piece of work, and it is not clear to me that the merge **will** achieve that goal.

**Comment by chaals** [comment]
Please note that a new CfC has been opened, on the proposal to overturn this decision: #1468 

Please respond to that CfC directly.
